src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -296,6 +304,11 @@ public static bool AreConnStringSetupForAE()
             return AEConnStrings.Count > 0 && IsNotAzureSynapse();
         }
 
+        public static bool IsSGXEnclaveConnStringSetup()
+        {
+            return !string.IsNullOrEmpty(TCPConnectionStringAASSGX);
```

Comment:
NIT: Methods in this class can use expression body.

Reply:
I changed this one but left the others. I don't want to add too many unrelated changes to the diff.